custom_components/tahoma/manifest.json

```diff
@@ -3,6 +3,8 @@
   "name": "Somfy TaHoma",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/tahoma",
-  "requirements": [],
+  "requirements": [
+    "git+https://github.com/iMicknl/python-tahoma-api.git#tahoma_api==0.1.0"
```

Comment:
Eventually this should be changed to our PyPi package, but for the time being this will work and take our latest master.

Reply:
OKi. I think, our lib is ready for a first public version.